guides/common/modules/proc_configuring-installation.adoc

```diff
@@ -12,6 +12,7 @@ If you lose connection to the shell where the installation command is running, s
 .Considerations
 
 * Use the `{installer-scenario} --help` command to display the available options and any default values.
+* For advanced installation options, use `{installer-scenario} --full-help`.
 If you do not specify any values, the default values are used.
```

Comment:
```suggestion
```
This used to be part of the first bullet item. You could move the line back to the first bullet but I recommend to drop it. (Default values are always used by default, no need to explain it.)